src/loadsave.cc

```diff
@@ -2548,23 +2546,6 @@ static bool_ do_savefile_aux(ls_flag_t flag)
 		do_fate(i, flag);
 	}
 
-	/* Load the Traps */
-	if (flag == ls_flag_t::SAVE) tmp16u = max_t_idx;
```

Comment:
It would be nice if we could keep save-file compat here by just reading/saving a bunch of 0's into the save file. (It would mean that max_t_idx would still be required -- or perhaps hardcoded if max_t_idx is the same for ToME and Theme.)

It's not a huge deal, I too break save-file compat all too often.